ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/data/CustomerCenterViewModel.kt

```diff
@@ -35,4 +37,16 @@ internal class CustomerCenterViewModelImpl(
         started = SharingStarted.WhileSubscribed(STOP_FLOW_TIMEOUT),
         initialValue = CustomerCenterState.Loading,
     )
+
+    private suspend fun loadPurchaseInformation(): PurchaseInformation? {
+        val customerInfo = purchases.awaitCustomerInfo(fetchPolicy = CacheFetchPolicy.FETCH_CURRENT)
+
+        // Customer Center WIP: udpate when we have subscription information in CustomerInfo
+        val activeEntitlement = customerInfo.entitlements.active.isEmpty()
+        if (activeEntitlement) {
+            return CustomerCenterConfigTestData.purchaseInformationMonthlyRenewing
```

Comment:
hardcoding this for now, will add the logic in a different PR. I need to add the new field to the `CustomerInfo`